projects/components/src/filter-bar/filter-bar.component.ts

```diff
@@ -95,32 +94,22 @@ export class FilterBarComponent implements OnChanges {
   ) {}
 
   public ngOnChanges(changes: TypedSimpleChanges<this>): void {
-    if (changes.scope) {
+    if (changes.attributes) {
       this.onFiltersChanged(this.filters || []);
-      this.subscribeToUrlFilters();
+      this.readFromUrlFilters();
```

Comment:
although existing logic, it feels like there's a lot of issues with this ngOnChanges block. It probably wasn't noticed because we don't generally change filter bar inputs. Line 98 for example - why would we emit a filter change when attributes have changed? readFromUrlFilters then calls onFiltersChanged on the next line, and may be called a second time on 103. onFiltersChanged is also called on 107, potentially emitting four times for a single change.